fs/config/configflags/configflags.go

```diff
@@ -87,6 +87,7 @@ func AddFlags(flagSet *pflag.FlagSet) {
 	flags.IntVarP(flagSet, &fs.Config.MaxBacklog, "max-backlog", "", fs.Config.MaxBacklog, "Maximum number of objects in sync or check backlog.")
 	flags.BoolVarP(flagSet, &fs.Config.StatsOneLine, "stats-one-line", "", fs.Config.StatsOneLine, "Make the stats fit on one line.")
 	flags.BoolVarP(flagSet, &fs.Config.Progress, "progress", "P", fs.Config.Progress, "Show progress during transfer.")
+	flags.BoolVarP(flagSet, &fs.Config.Cookie, "cookie", "", fs.Config.Cookie, "Enable session cookiejar.")
```

Comment:
I think my preference for the name of this flag would be "use-cookies"

Reply:
Actually I'm working on keeping cookies in an external cookiejar file as another option.
Now that rethinking about the naming convention, we could use `--use-cookies` to enable cookie support, and `--cookie-jar-file filename` to load and save cookies (which implies `--use-cookies`); Or, use `--cookies` to enable session cookie, and `--cookies filename` to enable cookie with loading and saving cookies.
How do you think?